### PR TITLE
Fix kt_android_local_test can't depend on testonly libraries

### PR DESCRIPTION
**Pull request by Bencodes**
https://github.com/bazelbuild/rules_kotlin/issues/536

### CODE REVIEW

File: kotlin/internal/jvm/android.bzl [hunk 1]
```diff
@@ -28,6 +28,7 @@ def _kt_android_artifact(
         kotlinc_opts = None,
         javac_opts = None,
         enable_data_binding = False,
+        testonly = False,
```
Comment (Kernald) [inline kotlin/internal/jvm/android.bzl:31]: Not a big deal, but I don't think adding this is needed - both the native `android.library` and the `_kt_jvm_library` fetch it from the `kwargs` anyway. The only change really needed here is the one on line 101.

Comment (Nikolas-LFDesigns) [reply kotlin/internal/jvm/android.bzl:31]: Tried the suggested fix to line 101, works as intended. Is there any way to speed up the process of applying these changes (like contributing to a subject PR)?

**Review by Nikolas-LFDesigns — CHANGES_REQUESTED** [review]

**Comment by Nikolas-LFDesigns** [comment]
@googlebot I fixed it.

**Comment by Nikolas-LFDesigns** [comment]
@cgruber @jin @restingbull @timpeut 
al checks passed, could you please check out the final solution?

**Review by cgruber — APPROVED** [review]
LGTM

**Pull request merged**